app/styles/_tables.less

```diff
@@ -151,8 +151,8 @@
   th#severity {width:10px;padding:0;}
   th#reason {width:150px;}
   th#message   {width:100%;} // ensures it gets remaining width
-  th#kind-name,th#message  {
-    .word-break;
```

Comment:
I was thinking we'd want it on all table cells. It only breaks mid-word if there is no natural break.